### PR TITLE
[DOC] - Updated pull request template to add new tests

### PR DESCRIPTION
**Pull request by Aarthy153**
#### Description
Updated the file `PULL_REQUEST_TEMPLATE.md` by adding a separate section for the contributors to add their additional tests. 

#### Issue reference
#4066 

#### Screenshot of the fix
image

### CODE REVIEW

**Comment by Aarthy153** [comment]
@miraep8 @fkiraly @GuzalBulatova requesting you to review this!

**Review by miraep8 — APPROVED** [review]
Looks good to me! Thanks for making this change!

**Review by fkiraly — CHANGES_REQUESTED** [review]
Looks good to me too!
Unfortunately linting (i.e., code formatting) is failing, here is the guidance how to ensure that code is formatted correctly:
https://www.sktime.org/en/latest/developer_guide/coding_standards.html
Let us know if you need help with that.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ If your contribution does add a new hard dependency, we may suggest to initially
 
 <!-- This section is particularly useful if you have a pull request that is still in development. You can guide the reviews to focus on the parts that are ready for their comments. We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->
 
+#### Did you add any tests rather than the exisiting ones?
```
Comment (fkiraly) [inline .github/PULL_REQUEST_TEMPLATE.md:31]: maybe we could shorten this, e.g., "Did you add any tests for the change?"

**Pull request closed without merging**